pkg/datapath/loader/loader.go

```diff
@@ -256,6 +260,15 @@ func removeObsoleteNetdevPrograms(devices []string) error {
 	ingressDevs := []netlink.Link{}
 	egressDevs := []netlink.Link{}
 	for _, l := range links {
+		// Remove old cil_from_netdev SKB program from cilium_wg0.
```

Comment:
This handles Cilium version upgrades... how to handle downgrades? The newly introduced `cil_from_wireguard` would still appear in the downgraded cluster, messing around with connectivity.